cpp/include/nvtx3/nvtx3.hpp

```diff
@@ -1783,38 +1783,38 @@ class event_attributes {
  *
  * Usage:
  * ```
- * nvtx3::domain_thread_range<> r0{"range 0"}; // Range in global domain
+ * nvtx3::scoped_range_in<> r0{"range 0"}; // Range in global domain
  *
- * nvtx3::thread_range r1{"range 1"}; // Alias for range in global domain
+ * nvtx3::scoped_range r1{"range 1"}; // Alias for range in global domain
```

Comment:
Maybe move this up to be the first example?  Putting the _in<> one first seems weird, since it's unlikely anyone will ever use that one explicitly.